netplugin/netd.go

```diff
@@ -181,8 +181,8 @@ func main() {
 	}
 
 	// Make sure we are running as root
-	usr, err := user.Current()
-	if (err != nil) || (usr.Username != "root") {
+	uid := syscall.Getuid()
```

Comment:
when netplugin runs as container, the UID = 0 may not mean the container has root privileges on the system. Is that a problem?